### DISCUSSION TITLE
mpv-install.bat 等文件的编码格式不是utf-8的，直接拖进vscode会乱码， 需要自行用GB 2312编码重新打开

### DISCUSSION THREAD

**Discussion by ameaninglessname** [hooke007/mpv_PlayKit#312] [mpv 通用]
![image](https://github.com/hooke007/MPV_lazy/assets/20263103/819d554b-f8c2-476d-a8fe-3967334af9c6)
是否要考虑改一下？

**Comment by hooke007** [comment]
这个文件用 utf-8 的意义是什么。
